Agilefreaks/FreaksCatalog-API
# [142] Create new projects query node

## Description
Create a new node to list all projects.

## Related

[[142] New "projects" query node](https://trello.com/c/gaMe9zcx/142-new-projects-query-node)

## Steps to Test or Reproduce
Go to Insomnia and execute this query:
```GraphQl
query {
    projects {
        name
        description
        logoUrl {
            uri
        }
        freaks {
            id
            firstName
        }
        technologies {
            name
            description
        }
    }
}
```
It should return a list of all `projects` from database, without pagination.
NOTE: data from projects table will be automatically added in database with help of _database seeds_.
